LF/character.js

```diff
@@ -386,6 +386,9 @@ define(['LF/livingobject', 'LF/global', 'core/combodec', 'core/util', 'LF/util']
           if ($.frame.N === 253) {
             $.trans.set_wait(0) // Woody's fly_crash
           }
+          if ($.frame.N >= 273 && $.frame.N <= 276 && $.name === 'Rudolf') {
```

Comment:
Thank you. But is it possible for us to identify some features of the data file rather than checking specifically the character's name?

Reply:
I have changed the $.name to $.id in the following commit